Project.toml

```diff
@@ -34,7 +34,7 @@ test = ["DataStructures", "DataValues", "Dates", "Logging", "Random", "Test"]
 [compat]
 julia = "1"
 CategoricalArrays = "0.7"
-Compat = "2, 3"
+Compat = "2.2, 3"
```

Comment:
`eachrow` and `eachcol` need Compat 2.2: https://github.com/JuliaLang/Compat.jl/releases/tag/v2.2.0